wasm/src/record/record_plaintext.rs

```diff
@@ -43,6 +43,17 @@ impl RecordPlaintext {
     pub fn gates(&self) -> u64 {
         ***self.0.gates()
     }
+
+    #[wasm_bindgen(js_name = serialNumber)]
+    pub fn serial_number(&self, private_key: &str, program_id: &str, record_name: &str) -> String {
+        let parsed_private_key = PrivateKeyNative::from_str(private_key).unwrap();
+        let parsed_program_id = ProgramIDNative::from_str(program_id).unwrap();
+        let record_identifier = IdentiferNative::from_str(record_name).unwrap();
+        let commitment = self.to_commitment(&parsed_program_id, &record_identifier).unwrap();
+
+        let serial_number = RecordPlaintextNative::serial_number(parsed_private_key, commitment).unwrap();
+        serial_number.to_string()
```

Comment:
```suggestion
        let parsed_program_id = ProgramIDNative::from_str(program_id).unwrap();
        let record_identifier = IdentiferNative::from_str(record_name).unwrap();
        let commitment = self.to_commitment(&parsed_program_id, &record_identifier).unwrap();

        let serial_number = RecordPlaintextNative::serial_number(private_key, commitment).unwrap();
        serial_number.to_string()
```

Reply:
Also a question - how is your application identifying spent records on the javascript/typescript side? Are you string matching against data being scanned from the API?

Reply:
Great question -- yes, we will need match serial number strings that we generate against serial number strings contained in the transaction outputs on the blockchain. We'll do that through our RPC's API, and we can either request all serial numbers on chain and then string match directly in the application, or we can send a list of specific serial number strings to the RPC to scan on our behalf.

Reply:
That is a reasonable way to do it given the block explorer exports those elements as plaintext. I am still thinking over whether or not this should have a newtype abstraction over it in the API. 

I think, for the purposes of future-proofing, this should probably be renamed to serialNumberString or a related name that indicates this is a string output instead of its own object.